### DISCUSSION TITLE
## Issue: Nylas Connector - Incremental Sync Parameter Injection Not Working as Expected

### DISCUSSION THREAD

**Discussion by saicharan9676** [airbytehq/airbyte#48487] [Connector Builder]
Hi Team,

### Connector Name
Source-Nylas

### Connector Version
[Please specify the version you're using]

### Description
The Nylas connector is not correctly injecting incremental sync parameters into the API request URL. Despite multiple configuration attempts, the URL is not being constructed as expected for incremental syncs.

### Current Behavior
The current URL being generated looks like this:
`https://api.us.nylas.com/v3/grants/2dd7fd4d-75da-4c23-8490-66a2fd26d4af/messages?search_query_native=220924800`

### Expected Behavior
The expected URL should look like this:
`https://api.us.nylas.com/v3/grants/2dd7fd4d-75da-4c23-8490-66a2fd26d4af/messages?search_query_native=from%3A+%22%7Bvisa.com%7D%22+after%3A+1234567890`

Where `1234567890` is the Unix timestamp for the start of the current slice.

### Steps to Reproduce
1. Configure the Nylas connector with incremental sync
2. Set up the `search_query_native` parameter to include both a dynamic parts (`from: "{domain.com}"`) and (`after: [timestamp]`)
3. Run a sync

We've tried various configurations including:
1. Using `start_time_option` with `inject_into: request_parameter`
2. Using custom parameter injection in the `parameters` section
3. Combining static and dynamic parts in the parameter value

None of these attempts have resulted in the correct URL construction.

The connector seems to be injecting a timestamp value into the `search_query_native` parameter, but it's not preserving the static part of the query or formatting it correctly.

**Comment by natikgadzhi** [comment]
The easiest fix is to fork it in Connector Builder and tweak incremental settings until it works, then contribute back the fix.

**Comment by saicharan9676** [comment]
Could you help me fixing the increment sync in the connector builder? I
have tried multiple approaches but none of them work.

Thanks,
Sai

[a comment by an automated account is not shown]
